doc/source/reference/arrays.classes.rst

```diff
@@ -284,12 +280,6 @@ NumPy provides several hooks that classes can customize:
    NumPy may also call this function without a context from non-ufuncs to
    allow preserving subclass information.
 
-   .. versionchanged:: 2.0
-      ``return_scalar`` is now passed as either ``False`` (usually) or ``True``
-      indicating that NumPy would return a scalar.
-      Subclasses may ignore the value, or return ``array[()]`` to behave more
-      like NumPy.
-
```

Comment:
Why removing this?

Reply:
The version changed is 2.0

Reply:
My bad, will check and revert this change.